src/main/kotlin/io/perfometer/statistics/consumer/StatisticsTextFileWriter.kt

```diff
@@ -0,0 +1,29 @@
+package io.perfometer.statistics.consumer
+
+import io.perfometer.statistics.ScenarioSummary
+import java.io.File
+import java.nio.charset.Charset
+import java.nio.file.Files
+import java.nio.file.Paths
+
+internal object StatisticsTextFileWriter {
+
+    fun write(scenarioSummary: ScenarioSummary) {
+        val formatted: String = StatisticsTextFormatter.format(scenarioSummary)
+        val file: File = createReportFile()
+        Files.write(file.toPath(), formatted.toByteArray(Charset.forName("UTF-8")))
+    }
+
+    private fun createReportFile(): File {
+        val file = Paths.get("${System.getProperty("user.dir")}/reports/report.txt").toFile()
```

Comment:
I think it would be better to use subdirectory (reports) of current directory, not user home. And would be nice if file name get higer probabilty of being unique, like adding current datetime suffix (eg report_20201028140410.txt)

Reply:
The name of the property is a bit misleading, it actually returns the current working dir of the application and not user's home directory.